XamarinCommunityToolkit/Views/AvatarView.shared.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.ComponentModel;
+using Xamarin.Forms;
+using static System.Math;
+
+namespace XamarinCommunityToolkit.Views
+{
+    public class AvatarView : Frame
```

Comment:
What's the reason to inherit from `Frame`, just the `CornerRadius` support? Since we're spending a bit of code on resetting some of the oddities within Frame and we probably don't want to support setting a `Content`.

Reply:
Yes, exactly for that reason.
The content property is re-defined and marked as obsolete + hidden for IntelliSense.

What's the different approach to achieving that?

Reply:
Yeah, I figured that was probably the case. I guess the only alternative is using some other view and doing the renderers for `CornerRadius` ourselves. That is a bit more effort than this probably needs though. Let's leave it as is, I was just curious :)

Reply:
> Yeah, I figured that was probably the case. I guess the only alternative is using some other view and doing the renderers for `CornerRadius` ourselves. That is a bit more effort than this probably needs though. Let's leave it as is, I was just curious :)

I hope we will use PancackeView once/if you submit PR xD